### DISCUSSION TITLE
An option disable some functions for other apps to handle those.

### DISCUSSION THREAD

**Discussion by MemoSainz** [seerge/g-helper#2734] [Ideas]
I got two examples. The first one, about the battery, if we want another app to handle the battery charge until 80%, we could have an option to disable it.

The second one.
As well, if we have a software to handle the gpu clock and more stuff of the gpu, as well the fans, to disable it in G-Helper in order to use another software like MSI Afterburner.

I also recommend another thing, for  to handle the lights. 
Like, if we disconnect the charger, for just show the lights of the keyboard, but it'll need more options perhaps. Options like this like the ones from Armoury Crate.

**Comment by seerge** [comment]
@MemoSainz hello,

For battery - if you also use MyASUS (and Asus Services) to limit charge limit - just set **same (80%)** charge limit in G-Helper and it will be fine.

For GPU clocks - by default app would not apply any core / memory clock offsets. If you had some overclocking in the app - click ``Factory Defaults`` it will reset settings, and won't be applying them until you drag sliders again.

For Backlight brigtnes - app remembers it separately per plugged in / on battery cases and switches automatically.

**Reply by MemoSainz** [reply]
Hello @seerge 

-In mine it doesn't seem to remember the light of the backlight. Wish there could be an option for me in that sense. 

-And about the battery, I wish an option to disable and use MyAsus for that task or another app in the future. Because when using G-Helper (even combined or not with MyAsus), when I turn off then turn on, the battery is being charged almost after I log in to windows. Like if it was from 80%, after turning on again it says 81%  and then it stops. So after three days later will say 85 or 89 and almost charging everything. 

The reason why I want MyAsus to take that task alone is for that. Because when using it, the led is always white and not orange, telling me that is not charging when is being turned on. Even if it's being turned on from off. Idk if it's also something that works with the BIOS but btw, that doesn't seem to happen to me with MyAsus.

In another good news. The app is doing the task so pretty well in most of aspects. It is also working with 60% of battery limit.
My laptop is an Asus Rog Strix G16 (G1614JV).

**Comment by seerge** [comment]
@MemoSainz it remembers brightness of backlight, controlled by ``FN+F2/F3``.

G-Helper and Asus Optimization Service set charge limit in the same (and only one possible way). It's just a single command that tells firmware to maintain charge, universal for all Asus laptops. 

When your laptop reboots it always resets all settings, including charge limit. So it will be charging until windows starts and then Asus Service or G-Helper sets limit. It's normal. If they both set same limit - effect is same.

If you really want prevent G-Helper from setting any limit - you can just delete ``charge_limit`` line from ``%appdata%\GHelper\config.json`` manually. But effect would be absolutely same - laptop will be charging during boot untill Asus service sets limit.
